src/main/java/games/strategy/util/Version.java

```diff
@@ -174,13 +179,13 @@ public String toStringFull(final String separator) {
     return toStringFull(separator, false);
   }
 
-  public String toStringFull(final String separator, final boolean noMicro) {
-    return m_major + separator + m_minor + separator + m_point + (noMicro ? "" : (separator + m_micro));
+  private String toStringFull(final String separator, final boolean noBuild) {
```

Comment:
This still can be simplified a lot no? Teh 'noBuild' variable is always false, why rename it and carry it forward when it can all be simplified?

Reply:
If/when the build is zero, noBuild becomes true. That will still happen when doing a "gradle run".

Reply:
Interesting, I may have missed that. It appeared to me the only place we call this method is on this line: `   return toStringFull(separator, false);`

Reply:
On line 189 of Version.java, method "toString"

Reply:
Actually, thinking about that case, we should set m_micro to Integer.MAX_VALUE in that case. That way you won't get the out of date version box if/when we move on the latest version.